### DISCUSSION TITLE
Trigger custom page view?

### DISCUSSION THREAD

**Discussion by derMarin** [plausible/analytics#1170] [Anything Else]
Hey,
is there a method to trigger a page view?

I'm launching a new website based on AJAX. So I don't have "real" page views.
Something like this in GA: https://developers.google.com/analytics/devguides/collection/analyticsjs/pages

I've found custom page view goals, but this is bad if you have 100+ pages: https://plausible.io/docs/pageview-goals

Thank you! :)

**Comment by ukutaht** [comment]
Hey,

If I understand the question correctly, you can trigger a pageview exactly like a custom event: https://plausible.io/docs/custom-event-goals. Pageviews are events with a special name: `pageview`. So once you've added the second line of the script tag like so:
```html
<script defer data-domain="<yourdomain.com>" src="https://plausible.io/js/plausible.js"></script>
<script>window.plausible = window.plausible || function() { (window.plausible.q = window.plausible.q || []).push(arguments) }</script>
```

You can then trigger a pageview as follows:

```javascript
plausible('pageview')
```

**Reply by derMarin** [reply]
Oh, okay! Now I got it! 
I'll give it a try! Thank you!
